tests/testthat/test-subgraph_centrality.R

```diff
@@ -0,0 +1,24 @@
+test_that("subgraph_centrality works", {
+  g <- make_graph("Frucht")
```

Comment:
I'll make a commit to rename those to not re-use names which is a pet peeve of mine :innocent: https://masalmon.eu/2024/05/23/refactoring-tests/

Reply:
I don't understand what you mean.

Reply:
In the test two objects were created with the name "g". I prefer the two names to be different.

Reply:
I see what you mean after looking at your commits, but keep in mind that there is no special significance to this being the Frucht graph here. If all you need is to not reassign `g`, I suggest `g1` and `g2` for similar situations in the future.

The reason why mathematicians don't spell out variable names is  not that they don't know any better. It's because sometimes it is useful to be generic and not be distracted by things that are irrelevant to the work at hand.

Reply:
Fair enough, but with g1 and g2 I'd easily get confused. They can be uninformative but they have to be really different for me :grin: